### PR TITLE
add e2e flow for leaderboard and dim table sorting

### PR DESCRIPTION
**Pull request by bcolloran**
adds e2e tests for dimension table and leaderboard sorting

* [Applications](?expand=1&template=applications_template.md)
* [Others](?expand=1&template=fallback_template.md)

### CODE REVIEW

**Comment by bcolloran** [comment]
will fix #3439

File: web-local/test/ui/dashboard-flows/leaderboard-and-dim-table-sort.spec.ts [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+        .locator("div")
+        .filter({ hasText: /^2$/ })
+        .getByRole("button", { name: "Filter dimension value" }),
+      page
+        .getByRole("table", { name: "Dimension table" })
+        .locator("div")
+        .filter({ hasText: /^-14$/ })
+        .getByRole("button", { name: "Filter dimension value" })
+    );
+    // sort by absolute change TWICE to sort by absolute change descending
```
Comment (ericpgreen2) [inline web-local/test/ui/dashboard-flows/leaderboard-and-dim-table-sort.spec.ts:189]: "ascending"?

File: web-local/test/ui/dashboard-flows/leaderboard-and-dim-table-sort.spec.ts [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        .filter({ hasText: /^-14$/ })
+        .getByRole("button", { name: "Filter dimension value" }),
+      page
+        .getByRole("table", { name: "Dimension table" })
+        .locator("div")
+        .filter({ hasText: /^2$/ })
+        .getByRole("button", { name: "Filter dimension value" })
+    );
+
+    // sort by pct change ONCE to sort by pct change ascending
```
Comment (ericpgreen2) [inline web-local/test/ui/dashboard-flows/leaderboard-and-dim-table-sort.spec.ts:204]: "descending"?

File: web-local/test/ui/dashboard-flows/leaderboard-and-dim-table-sort.spec.ts [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+        .locator("div")
+        .filter({ hasText: /^4%$/ })
+        .getByRole("button", { name: "Filter dimension value" }),
+      page
+        .locator("div")
+        .filter({ hasText: /^3%$/ })
+        .getByRole("button", { name: "Filter dimension value" })
+    );
+
+    // sort by pct change TWICE to sort by pct change descending
```
Comment (ericpgreen2) [inline web-local/test/ui/dashboard-flows/leaderboard-and-dim-table-sort.spec.ts:217]: "ascending"?

File: web-local/test/ui/dashboard-flows/leaderboard-and-dim-table-sort.spec.ts [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+        .locator("div")
+        .filter({ hasText: /^3%$/ })
+        .getByRole("button", { name: "Filter dimension value" }),
+      page
+        .locator("div")
+        .filter({ hasText: /^4%$/ })
+        .getByRole("button", { name: "Filter dimension value" })
+    );
+
+    // sort by pct of total ONCE to sort by pct of total ascending
```
Comment (ericpgreen2) [inline web-local/test/ui/dashboard-flows/leaderboard-and-dim-table-sort.spec.ts:230]: "descending"?

File: web-local/test/ui/dashboard-flows/leaderboard-and-dim-table-sort.spec.ts [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+      page
+        .locator("div")
+        .filter({ hasText: /^34%$/ })
+        .getByRole("button", { name: "Filter dimension value" }),
+      page
+        .locator("div")
+        .filter({ hasText: /^25%$/ })
+        .getByRole("button", { name: "Filter dimension value" })
+    );
+    // sort by pct of total TWICE to sort by pct of total descending
```
Comment (ericpgreen2) [inline web-local/test/ui/dashboard-flows/leaderboard-and-dim-table-sort.spec.ts:242]: "ascending"?

**Review by ericpgreen2 — APPROVED** [review]
Great 👍

**Comment by bcolloran** [comment]
Thanks @ericpgreen2 , nice catches on those typos! Fixed! :-)

**Pull request merged**